common/lib/capa/capa/templates/codeinput.html

```diff
@@ -4,7 +4,11 @@
 from openedx.core.djangolib.markup import HTML
 %>
 <div id="textbox_${id}" class="capa_inputtype textbox cminput">
-  <label class="sr problem-group-label" for="cm-textarea-${id}">Code Editor ${response_data['label']}</label>
+  % if response_data['label']:
+    <label class="problem-group-label" for="cm-textarea-${id}">${response_data['label']}</label>
+  % else:
+    <label class="sr problem-group-label" for="cm-textarea-${id}">${_('Code Editor')}</label>
+  % endif
```

Comment:
Note: This will respect the displayed text as it is and include SR text when it is not available.